src/recommendations/data/tests/hooks.test.jsx

```diff
@@ -17,6 +19,23 @@ jest.mock('../../../data/algolia', () => ({
 
 jest.mock('../algoliaResultsParser', () => jest.fn((course) => course));
 
+const renderHook = (hookCallback) => {
+  const result = {
+    current: null,
+  };
+
+  const Component = () => {
+    const val = hookCallback();
+    React.useEffect(() => {
+      result.current = val;
+    });
+    return null;
+  };
+
+  render(<Component />);
+  return { result };
+};
+
```

Comment:
I had to remove `@testing-library/react-hooks` due to package conflicts, so I've replaced it with this simplified hook testing code. In later versions this hook is part of the react testing library so we can pull it from there and remove this.